deployer/__main__.py

```diff
@@ -355,6 +391,7 @@ def main():
         )
     elif args.action == "validate":
         validate_cluster_config(args.cluster_name)
+        validate_support_config(args.cluster_name)
```

Comment:
Seeing these three `validate_` functions makes me happy :-)

Reply:
Only useful if they work though 😉 https://github.com/2i2c-org/infrastructure/pull/1048#discussion_r817946077

Reply:
@sgibson91 we just need `validate_validation_config()` as well :D #1049